tool/disable_ipv6.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+sysctl -w net.ipv6.conf.all.disable_ipv6=1
+sysctl -w net.ipv6.conf.default.disable_ipv6=1
+sysctl -w net.ipv6.conf.lo.disable_ipv6=1
+
+cat /etc/hosts
+ruby -e "hosts = File.read('/etc/hosts').sub(/^::1\s*localhost.*$/, ''); File.write('/etc/hosts', hosts)"
+cat /etc/hosts
+
+cat /etc/services || ruby -e "File.write('/etc/services', 'ssh 22/tcp')"
```

Comment:
@fkorotkov I noticed the `tool/disable_ipv6.sh` you added on the PR includes this line `at /etc/services || ruby -e "File.write('/etc/services', 'ssh 22/tcp')"` unlike the file used in the past: https://github.com/ruby/ruby/commit/86ca640287eccc8426b87903cac1ec428d7ede8b . Why is this line needed?

Reply:
There is the `/etc/services` in the container image. It seems the `cat /etc/services` is always succeeded.

```
$ podman run --rm -it ghcr.io/ruby/ruby-ci-image:clang-12 bash
root@41935005e08b:/# ls /etc/services 
/etc/services
```

Reply:
It might be an artifact from the time before we created the Arm images under https://github.com/ruby/ruby-ci-image. If I recall correctly there were some network tests that needed it. I think it will be safe to delete it.

Reply:
Yeah, tests are working without the line https://cirrus-ci.com/build/4940483613949952

Reply:
Thanks for checking. I will delete the line.